components/wasm-opt/tests/api_tests.rs

```diff
@@ -97,6 +93,7 @@ fn test_optimization_options_os() -> anyhow::Result<()> {
     Ok(())
 }
 
+// TODO: Figure out how to fix, or if this is still even needed
 #[test]
 fn optimization_read_module_error_works() -> anyhow::Result<()> {
```

Comment:
The result being tested in the assert no longer errors. I don't know if this is because I made a mistake doing the upgrade, or if it's due to a change in the unerlying C++ code in the new version.

Comparing `PassRunner::run` between [version_116](https://github.com/WebAssembly/binaryen/blob/11dba9b1c2ad988500b329727f39f4d8786918c5/src/passes/pass.cpp#L703-L706) and [version_121](https://github.com/WebAssembly/binaryen/blob/353b759b230dff8fb82aeb157aeb6db360d74a49/src/passes/pass.cpp#L817), there is only one difference: the former does an assert at the start of the function and the latter doesn't.